Source/Core/VideoBackends/OGL/FramebufferManager.cpp

```diff
@@ -263,23 +263,27 @@ FramebufferManager::FramebufferManager(int targetWidth, int targetHeight, int ms
                 "vec4 sampleEFB(ivec3 pos) {\n"
                 "	vec4 color = vec4(0.0, 0.0, 0.0, 0.0);\n"
                 "	for(int i=0; i<" +
-                samples.str() + "; i++)\n"
-                                "		color += texelFetch(samp9, pos, i);\n"
-                                "	return color / " +
-                samples.str() + ";\n"
-                                "}\n";
+                samples.str() +
+                "; i++)\n"
```

Comment:
This one was already mangled by a previous formatting, but should probally be fixed too.